lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1098,6 +1098,15 @@
     @include padding($baseline $baseline 0 $baseline);
     border: 1px solid $gray-l5;
   }
+
+  .padded-content {
+    padding-left: $baseline !important;
+    padding-right: $baseline !important;
```

Comment:
Woah! Why the ``!important``? Any way you can scope this with a more descriptive selector or its order in the cascade?